### ISSUE TITLE
@swc-node/register@1.6.8 don't have exports from master branch

### ISSUE THREAD

**Issue by krutoo** [swc-project/swc-node#749]
I just do everything accrding to readme.md:

```
npm i -D @swc-node/register

node --import @swc-node/register/esm-register scripts/prebuild.ts
```

And I get an error:
```
node:internal/modules/esm/resolve:303
  return new ERR_PACKAGE_PATH_NOT_EXPORTED(
         ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './esm-register' is not defined by "exports" in /home/petrov-dm/Projects/some-project/node_modules/@swc-node/register/package.json imported from /home/petrov-dm/Projects/some-project/
```

details:
```
node -v
v20.11.0

npm pkg get type
"module"
```

I removed `package-lock.json` and `node_modules` and install `swc-node/register` - it not helps

---

I see in node_modules there is no `/esm-register` in exports field.

But in master branch there is:
https://github.com/swc-project/swc-node/blob/master/packages/register/package.json#L79

**Comment by yeliex** [comment]
this commit just merged into the master branch and has not been published yet

waiting @Brooooooklyn to publish. or you can use https://npmjs.com/swc-register-esm temporary

**Comment by yeliex** [comment]
this commit just merged into the master branch and has not been published yet

waiting @Brooooooklyn to publish. or you can use https://npmjs.com/swc-register-esm temporary

**Comment by shifenhutu** [comment]
same here

for temporary

i use this :
`node --loader @swc-node/register/esm script.ts `

it from  https://github.com/swc-project/swc-node/issues/704#issuecomment-1502046521

**Comment by yeliex** [comment]
this issue could be closed cause @swc-node/register@1.8.0 has released

test with new version and create another issue if has new problem
